lib/config/validation.spec.ts

```diff
@@ -309,178 +309,191 @@ describe('config/validation', () => {
       expect(errors).toMatchSnapshot();
     });
 
-    it('validates regEx for each fileMatch', async () => {
-      const config = {
-        regexManagers: [
+    describe('customManagers/regex', () => {
+      it('validates regEx for each fileMatch', async () => {
+        const config = {
+          customManagers: [
+            {
+              fileMatch: ['js', '***$}{]]['],
+              matchStrings: ['^(?<depName>foo)(?<currentValue>bar)$'],
+              datasourceTemplate: 'maven',
+              versioningTemplate: 'gradle',
+            },
+          ],
+        };
+        const { warnings, errors } = await configValidation.validateConfig(
+          config,
+          true
+        );
+        expect(warnings).toHaveLength(0);
+        expect(errors).toHaveLength(1);
+        expect(errors).toMatchObject([
```

Comment:
does toEqual work too or do we have more properties to ignore for the test?

Reply:
`toEqual` doesn't care about array's length, whereas `toMatchObject` does. So using it and removing the 
`toHaveLength` assestions.

Reply:
it works